src/_includes/episodes_layout.njk

```diff
@@ -22,8 +22,12 @@
           Published {{ publish_date | formatDate }} -
           <a href="{{ spotify_link }}" class="text-primary-100 dark:text-secondary-100 hover:underline">Listen on your favourite podcast player</a>
         </p>
+        <div class="aspect-w-16 aspect-h-9">
```

Comment:
do we need the tailwind aspect plugin at all?

can't we just use `aspect-video` or `aspect-[16/9]`?

https://tailwindcss.com/docs/aspect-ratio#setting-the-aspect-ratio